move-mutation-test/src/cli.rs

```diff
@@ -65,27 +65,30 @@ pub fn create_mutator_options(
 // Info: this set struct is based on TestPackage in `aptos-core/crates/aptos/src/move_tool/mod.rs`.
 #[derive(Parser, Debug, Clone)]
 pub struct TestBuildConfig {
-    /// Options for compiling a move package dir.
-    // We might move some options out and have our own option struct here - not all options are
-    // needed for mutation testing.
-    #[clap(flatten)]
-    pub move_pkg: MovePackageDir,
-
-    /// Dump storage state on failure.
-    #[clap(long = "dump")]
-    pub dump_state: bool,
-
-    /// A filter string to determine which unit tests to run.
-    #[clap(long)]
+    /// A filter string to determine which unit tests to run
+    #[clap(long, short)]
     pub filter: Option<String>,
 
     /// A boolean value to skip warnings.
     #[clap(long)]
     pub ignore_compile_warnings: bool,
 
-    /// Compute and then use unit test computed coverage to generate mutants only for covered code.
-    #[clap(long = "coverage", conflicts_with = "use_generated_mutants")]
-    pub apply_coverage: bool,
+    #[clap(flatten)]
+    pub move_options: MovePackageOptions,
+
+    /// The maximum number of instructions that can be executed by a test
+    ///
+    /// If set, the number of instructions executed by one test will be bounded
+    #[clap(long = "instructions", default_value_t = 100000)]
+    pub instruction_execution_bound: u64,
```

Comment:
Let's remove this, this is unused - I just created a PR to officially clean it up:
https://github.com/aptos-labs/aptos-core/pull/17513